data/transition-sites/dft_bluebadge.yml

```diff
@@ -0,0 +1,6 @@
+---
+site: dft_bluebadge
+whitehall_slug: department-for-transport
+homepage: https://www.gov.uk/government/organisations/department-for-transport
+tna_timestamp: 20201010101010
+host: bluebadge.direct.gov.uk
```

Comment:
I think we should also want an aliases entry here:
```
aliases:
  - bluebadge2.direct.gov.uk
  - www.bluebadge.direct.gov.uk
```
@thomasleese, do I understand that correctly?

Reply:
@schmie Ah yeah, I think you're right about that.